src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type MovieType = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<MovieType> = ({ movie }) => {
```

Comment:
you can destructure movie 
```suggestion
export const MovieCard: React.FC<MovieType> = ({ movie }) => {
  const { 
    title,
    description, 
    imgUrl, 
    imdbUrl 
   } = movie;
```